nncf/parameters.py

```diff
@@ -33,12 +33,23 @@ class TargetDevice(Enum):
     VPU = 'VPU'
 
 
+class ModelType(Enum):
+    """
+    Describes the model type the specificity of which will be taken into
+    account during compression.
+
+    :param TRANSFORMER: Transformer-based models
+        (https://arxiv.org/pdf/1706.03762.pdf)
+    """
+    TRANSFORMER = 'transformer'
```

Comment:
Could you please use this class here?
https://github.com/openvinotoolkit/nncf/blob/00dd76828eb7dc58717e95f2c9330a9fd36ba670/examples/experimental/openvino/bert/main.py#L72

Reply:
I will do but, please note, that I cannot test these changes.